flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackend.java

```diff
@@ -104,7 +105,8 @@
 	private static final Logger LOG = LoggerFactory.getLogger(RocksDBStateBackend.class);
 
 	/** The number of (re)tries for loading the RocksDB JNI library. */
-	private static final int ROCKSDB_LIB_LOADING_ATTEMPTS = 3;
+	@VisibleForTesting
```

Comment:
This seems not used in the tests any more, so we can change the visibility back.